site/en/blog/whats-new-in-web-on-android-io2023/index.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What’s new for web on Android 2023
+description: Updates to tools that bring the web to Android. Privacy enhancements and better support for large screens in WebView, partial custom tabs in Custom Tabs, easier installation and integrated features for PWA, Play Store billing management for TWA and more.
+date: 2023-05-10
+authors:
+  - ajara
+  - sebastianbenz
+tags:
+  - capabilities
+  - progressive-web-apps
+  - android
+hero: 
```

Comment:
Can we get at least a thumbnail image? The listing looks better with them. Use thumbnail rather than hero and it won't display above the article.

Reply:
I left it out as we thought that we'd be using the video as hero. But I can also add the video thumbnail as hero.

Reply:
yeah, or just add it as a thumbnail, that way it will show on the listing page but not on the article.

Reply:
added the video thumbnail